documentation/ame-guide/modules/ROOT/pages/modeling/event.adoc

```diff
@@ -8,28 +8,26 @@ Assets can for instance emit events to notify other assets in case of special oc
 
 To add an Event, proceed as follows:
 
-* Open the Workspace sidebar
-* Drag the Event element and drop it to the graph area canvas
+* Open the Workspace sidebar.
+* Drag the Event element and drop it to the graph area canvas.
 
 == Connect the event with the Aspect
 For a new model that only has an Aspect so far:
 
-* Click the Event
-* Right-click the Event to open the context menu
-** Click the Connect with ...
-** Select the Aspect element
-** Confirm with Connect
+* Click the Event.
```

Comment:
is this necessary? right-click is enough?

Reply:
@h-auch13 yes, right click is enough to select an element and open the context menu. 
@aniluede For me it works fine